ocs_ci/ocs/node.py

```diff
@@ -1073,3 +1080,18 @@ def generate_node_names_for_vsphere(count, prefix="compute-"):
         f"{prefix}{current_compute_suffix + node_count}"
         for node_count in range(1, count + 1)
     ]
+
+
+def is_ocs_labeled(node_name):
```

Comment:
consider making this function generic to any passed label. It would get the node name and the label to check

Reply:
Good idea. I changed the function to be generic.